gerousiabot/bot_handlers.py

```diff
@@ -13,37 +18,52 @@ def get_bot_handlers() -> list:
     """
     start_handler = CommandHandler('start', start)
     user_query_handler = CommandHandler('chkusers', check_users)
+    help_handler = CommandHandler('help', help_command)
+
     button_handler = CallbackQueryHandler(keyboard_button_pressed)
     unknown_handler = MessageHandler(Filters.command, unknown_command)
 
     # unknown command handler must be the last added handler or we run into logical errors
-    return [start_handler, button_handler, user_query_handler, unknown_handler]
+    return [start_handler, user_query_handler, help_handler, button_handler, unknown_handler]
+
+
+def start(update, context) -> None:
+    """Handler definition for the bot start command: /start"""
+    context.bot.send_message(chat_id=update.effective_chat.id, text="Gerousia bot! 🤖")
+    kb_markup = InlineKeyboardMarkup(get_keyboard())
+    update.message.reply_text('Please select one:', reply_markup=kb_markup)
 
 
-def start_keyboard() -> list:
+def get_keyboard() -> list:
     """Initialize the keyboard"""
-    keyboard = [InlineKeyboardButton("Check Users", callback_data=Response.CHECK_USERS),
-                InlineKeyboardButton("Blank button", callback_data=Response.BLANK)]
+    keyboard = [[
```

Comment:
[Focus Point] This is what caused the bug. `InlineKeyboardMarkup()` expected a 2D array